pkg/commands/git_commands/commit_loader.go

```diff
@@ -328,6 +339,84 @@ func (self *CommitLoader) getInteractiveRebasingCommits() ([]*models.Commit, err
 	return commits, nil
 }
 
+func (self *CommitLoader) getConflictedCommit(todos []todo.Todo) string {
```

Comment:
There's a lot of conditional logic in here, and it's a pure function of three things (todos, rebase-merge/done, rebase-merge/amend). What are your thoughts on adding unit tests for this?

Reply:
I started writing them, and I could continue if you really want to have them, but I'm not convinced of their value. We could synthesize the inputs in the same way as git does it in the various user scenarios, and then check that we get the expected result, but nobody knows if those inputs are actually what git produces in the respective scenarios. I find it much more valuable to test the actual user scenarios with integration tests, and I did try to be comprehensive with that in this branch.

Reply:
OK, I finished the tests: ec13dd22d355. Still not convinced they are very valuable, but since I wrote them already, we might as well keep them I suppose.